spec/scrabble_spec.rb

```diff
@@ -0,0 +1,95 @@
+require './lib/scrabble'
+
+describe Scrabble::Scrabble do
+
+  describe "Breakfast: Scrabble#score" do
+
+    it "contains a self.score method" do
+      expect(Scrabble::Scrabble).to respond_to :score
+    end
+
+    [
+      [5, "cat"],
+      [12, "kitty"],
+      [10, "rabbit"],
+      [10, "bunny"],
+      [9, "fly"],
+      [9, "unicorn"],
+      [5, "dog"]
+    ].each do |value, word|
+      it "returns #{value} for #{word}" do
```

Comment:
I love this approach to testing a bunch of words, but in separate tests so that you can see failures for each word. Nice!